### PR TITLE
remove code from corehq.__init__

### PR DESCRIPTION
**Pull request by snopoke**
@TylerSheffels

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/dashboard/models.py, line 2 - F401 'Domain' imported but unused
* corehq/apps/hqadmin/reports.py, line 5 - F401 'Domain' imported but unused
* corehq/apps/reports/filters/fixtures.py, line 4 - F401 'Domain' imported but unused
* corehq/apps/smsbillables/filters.py, line 1 - F401 'Domain' imported but unused
* corehq/reports.py, line 161 - E126 continuation line over-indented for hanging indent
* corehq/reports.py, line 163 - E121 continuation line under-indented for hanging indent
* corehq/tabs.py, line 13 - E261 at least two spaces before inline comment
* corehq/tabs.py, line 30 - E261 at least two spaces before inline comment
* custom/ewsghana/utils.py, line 3 - F401 'Domain' imported but unused

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/dashboard/models.py, line 2 - F401 'Domain' imported but unused
* corehq/apps/hqadmin/reports.py, line 5 - F401 'Domain' imported but unused
* corehq/apps/reports/filters/fixtures.py, line 4 - F401 'Domain' imported but unused
* corehq/apps/smsbillables/filters.py, line 1 - F401 'Domain' imported but unused
* corehq/reports.py, line 161 - E126 continuation line over-indented for hanging indent
* corehq/reports.py, line 163 - E121 continuation line under-indented for hanging indent
* corehq/tabs.py, line 13 - E261 at least two spaces before inline comment
* corehq/tabs.py, line 30 - E261 at least two spaces before inline comment
* custom/ewsghana/utils.py, line 3 - F401 'Domain' imported but unused

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/dashboard/models.py, line 2 - F401 'Domain' imported but unused
* corehq/apps/hqadmin/reports.py, line 5 - F401 'Domain' imported but unused
* corehq/apps/reports/filters/fixtures.py, line 4 - F401 'Domain' imported but unused
* corehq/apps/smsbillables/filters.py, line 1 - F401 'Domain' imported but unused
* corehq/reports.py, line 161 - E126 continuation line over-indented for hanging indent
* corehq/reports.py, line 163 - E121 continuation line under-indented for hanging indent
* corehq/tabs.py, line 13 - E261 at least two spaces before inline comment
* corehq/tabs.py, line 30 - E261 at least two spaces before inline comment
* custom/ewsghana/utils.py, line 3 - F401 'Domain' imported but unused

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/dashboard/models.py, line 2 - F401 'Domain' imported but unused
* corehq/apps/hqadmin/reports.py, line 5 - F401 'Domain' imported but unused
* corehq/apps/reports/filters/fixtures.py, line 4 - F401 'Domain' imported but unused
* corehq/apps/smsbillables/filters.py, line 1 - F401 'Domain' imported but unused
* corehq/reports.py, line 161 - E126 continuation line over-indented for hanging indent
* corehq/reports.py, line 163 - E121 continuation line under-indented for hanging indent
* corehq/tabs.py, line 13 - E261 at least two spaces before inline comment
* corehq/tabs.py, line 30 - E261 at least two spaces before inline comment
* custom/ewsghana/utils.py, line 3 - F401 'Domain' imported but unused

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/dashboard/models.py, line 2 - F401 'Domain' imported but unused
* corehq/apps/hqadmin/reports.py, line 5 - F401 'Domain' imported but unused
* corehq/apps/reports/filters/fixtures.py, line 4 - F401 'Domain' imported but unused
* corehq/apps/smsbillables/filters.py, line 1 - F401 'Domain' imported but unused
* corehq/reports.py, line 161 - E126 continuation line over-indented for hanging indent
* corehq/reports.py, line 163 - E121 continuation line under-indented for hanging indent
* corehq/tabs.py, line 13 - E261 at least two spaces before inline comment
* corehq/tabs.py, line 30 - E261 at least two spaces before inline comment
* custom/ewsghana/utils.py, line 3 - F401 'Domain' imported but unused

File: corehq/apps/smsbillables/filters.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from corehq import Domain
+from corehq.apps.domain.models import Domain
```
Comment (dimagimon) [inline corehq/apps/smsbillables/filters.py]: F401 'Domain' imported but unused

**Comment by snopoke** [comment]
@dannyroberts

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by czue** [comment]
awesome stuff

**Pull request merged**